examples/playlist/README.md

```diff
@@ -0,0 +1,57 @@
+# Playlist examples
```

Comment:
I know the general agreement is to go away from Thema and use _santhosys_ or anything else based on JSON Schema. But, considering the purpose here seems to be demonstrating the benefits of the generic interface, and that it looks like we may need to keep compatibility for both for a while, wouldn't make more sense, and make it more readable, to have each example isolated?

I'd be happy to help with that if that makes sense and does worth it!

Reply:
excellent -- yes this exists because we need *something* to exercise interfaces so we can learn if they are reasonable.  This is the part I expect will evolve the most